ledger/indexer-benchmark/README.md

```diff
@@ -1,7 +1,3 @@
 # `indexer-benchmark`
 
-A tool that measures the performance of the indexer.
-
-# Input format
-
-This tool consumes kvutils ledger export files.
+A tool that measures the performance of the indexer. Used in downstream repositories.
```

Comment:
It cannot be used if it is removed from `artifacts.yaml`

Reply:
Oh, right! And before we were even using a fat jar for a library dependency for the other repo.
Let me try to fix that.
Thanks for the headsup :+1: